app/controllers/oauth2_controller.rb

```diff
@@ -16,6 +16,15 @@ def create
     when 'google_auth_code'
       authenticate_with_google(params[:auth_code])
     else
+      other_authenticators
```

Comment:
Can you disable that check for this specific method. Splitting it up like this might make Rubocop happy but actually makes the code much worse ;)